Description: Link to the same version of the source code as what's installed
Author: Ximin Luo <infinity0@debian.org>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/tools/nimweb.nim
+++ b/tools/nimweb.nim
@@ -60,7 +60,7 @@
   c.ticker = ""
   c.vars = newStringTable(modeStyleInsensitive)
   c.gitRepo = "https://github.com/nim-lang/Nim/tree"
-  c.gitCommit = "master"
+  c.gitCommit = "v" & system.NimVersion
   c.numProcessors = countProcessors()
   # Attempts to obtain the git current commit.
   when false:
